section2/exercises/if-statements.js

```diff
@@ -31,19 +31,27 @@ if (weather === 'sunny') {
   console.log('good to go!');
 }
 
-/* 
+/*
 YOU DO:
-Using the dogAge variable defined below, 
-determine if a dog is a puppy (2 or younger), 
+Using the dogAge variable defined below,
+determine if a dog is a puppy (2 or younger),
 an adult, or elderly (10 or older).
 
 Log to the console the appropriate age range
 (puppy, adult, elderly).
 */
 
-var dogAge = 3;
+var dogAge = 12;
 // Write your conditional here
 
+if (dogAge <= 2) {
+  console.log('puppy');
+} else if (dogAge <= 9) {
+    console.log('adult');
```

Comment:
Be sure to keep your indentation uniform, this one looks like it may be double tabbed.